### DISCUSSION TITLE
API

### DISCUSSION THREAD

**Discussion by THEOEHT03** [MadeInPierre/finalynx#52] [General]
Bonjour je cherche a afficher mon portefeuille sur Better Touch Bar ( app pour afficher sur la touch bar macbook apple ) je n'arrive pas a generer mon api pour pouvoir faire un script pour afficher cela

**Comment by MadeInPierre** [comment]
Bonjour, je ne connais pas l'écosystème Mac, j'imagine qu'il y a des tutoriels pour contrôler la touch bar avec un script Python :) Pour l'API, j'aurais besoin de plus de détails pour pouvoir aider. Le fichier [partagé ici](https://github.com/MadeInPierre/finalynx/blob/main/finalynx/fetch/fetch_finary.py) est le meilleur exemple que je puisse donner, il devrait avoir tout ce qu'il faut.

**Reply by THEOEHT03** [reply]
je cherche a avoir mon api pour afficher juste la ligne de mon montant de portefeuille
